draft-ietf-mls-protocol.md

```diff
@@ -3818,8 +3818,13 @@ struct {
 } GroupInfo;
 ~~~
 
-New members MUST verify the `signature` using the public key taken from the
-credential in the leaf node of the ratchet tree with leaf index `signer`. The
+New members MUST verify that `group_id` is unique among the groups they're
+currently participating in. The Delivery Service MAY attempt to ensure that
```

Comment:
I may be over-thinking this, but: The group ID isn't necessarily exposed to the DS at group creation time, since the act that creates the group is a Welcome message, which encrypts the group ID.  The group ID is visible in MLSPlaintext and MLSCiphertext, though, so the DS can pick it up there.